zppy/templates/default.ini

```diff
@@ -1,4 +1,5 @@
 [default]
+active = boolean(default=False)
```

Comment:
Flipping the default to `False` is an incompatible API change because users may currently be running tasks without `active = True` specified (since it was in the `default.ini` file).